pangolin/4.3-pdata-1.20/README.md

```diff
@@ -0,0 +1,47 @@
+# pangolin docker image
+
+Main tool : [pangolin](https://github.com/cov-lineages/pangolin)
+
+Full documentation: [https://cov-lineages.org/resources/pangolin.html](https://cov-lineages.org/resources/pangolin.html)
+
+Phylogenetic Assignment of Named Global Outbreak LINeages
+
+Additional tools:
+
+- [pangolin-data](https://github.com/cov-lineages/pangolin-data) 1.20
+- [pangolin-assignment](https://github.com/cov-lineages/pangolin-assignment)  1.20
+- [minimap2](https://github.com/lh3/minimap2) 2.26-r1175
+- [usher](https://github.com/yatisht/usher) 0.6.2
+- [faToVcf](https://github.com/yatisht/usher) 426
+- [scorpio](https://github.com/cov-lineages/scorpio) 0.3.17
+- [constellations](https://github.com/cov-lineages/constellations) 0.1.10
+- [gofasta](https://github.com/virus-evolution/gofasta) 1.2.0
+- [mafft](https://mafft.cbrc.jp/alignment/software/) 7.520
+- python 3.8.15
+
+## pangoLEARN deprecation
+
+This docker image contains `pangolin-data` v1.20. The pangoLEARN model has not been updated since pangolin-data version 1.18. Only the the underlying UShER tree/protobuf file will be maintained for the forseeable future. **Please use the UShER mode of pangolin if you want to stay up-to-date with the most recent lineages.** [See pangolin-data release notes here for more details](https://github.com/cov-lineages/pangolin-data/releases/tag/v1.20)
```

Comment:
Are you referring to the main `/README.md`? 

I have started adding relative links on the main `/README.md` that links the user to the pangolin-specific docs at `pangolin/4.3-pdata-1.20/README.md` which states that pangoLEARN has been deprecated

Reply:
Just the tool specific one.

I think the following text suggests that pangolearn is still an option, even if not the best one.

> This docker image contains `pangolin-data` v1.20. The pangoLEARN model has not been updated since pangolin-data version 1.18. Only the the underlying UShER tree/protobuf file will be maintained for the forseeable future. **Please use the UShER mode of pangolin if you want to stay up-to-date with the most recent lineages.** [See pangolin-data release notes here for more details](https://github.com/cov-lineages/pangolin-data/releases/tag/v1.20)

Reply:
OK, I will clarify the text.

Reply:
It is still an option, but I clarified what happens when the pangolearn mode is used. Hopefully the updated readme is more clear now

Reply:
I like this better. Once the tests pass, I'll merge this PR and push to dockerhub and quay